pages/home/auth/call-third-party-apis-directly.mdx

```diff
@@ -19,8 +20,14 @@ This can be useful when you need to manage authorization flows in your applicati
 
 ### Prerequisites
 
-- [Set up Arcade](/home/quickstart)
-- Install the Arcade Client and the required libraries:
+- Sign up for an <SignupLink utmParams={{
+            utm_source: "docs",
+            utm_medium: "page",
+            utm_campaign: "custom-tools",
```

Comment:
probably want a different string here? seems like this isn't the custom tools page

Reply:
fixed